contrib/competitions/smt-comp/run-script-smtcomp-current

```diff
@@ -56,7 +56,8 @@ QF_LIA)
   finishwith --miplib-trick --miplib-trick-subs=4 --use-approx --lemmas-on-replay-failure --replay-early-close-depth=4 --replay-lemma-reject-cut=128 --replay-reject-cut=512 --unconstrained-simp --use-soi --pb-rewrites --ite-simp --simp-ite-compress
   ;;
 QF_NIA)
-  trywith 480 --nl-ext-tplanes --decision=internal
+  trywith 420 --nl-ext-tplanes --decision=justification
```

Comment:
The third line of the strategy is a duplicate, either remove it, or perhaps change it to `--decision=justification-old`. Nevertheless, we should deallocate 60 seconds from somewhere else, say this line 480 -> 420.

Reply:
Argh, good catch. I've updated it.